plugins/obs-webrtc/whip-output.cpp

```diff
@@ -368,6 +424,24 @@ bool WHIPOutput::Connect()
 	CURLU *url_builder = curl_url();
 	auto last_location_header = location_headers.back();
 
+	struct curl_header *prev = nullptr;
```

Comment:
Probably want a comment here saying that the link header parsing is happening here.

Reply:
@DDRBoxman Fixed! Added a comment explaining what is happening